package.json

```diff
@@ -4,31 +4,34 @@
   "description": "analytics service for lena",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node server/index.js",
```

Comment:
If filename is index.js, can simplify to "node server" instead for more concise code